GOVERNANCE.md

```diff
@@ -1,130 +1,125 @@
+Responsibilities:
 
-### Oversight Committee
+- Primary role of Chairs is governance of the Working Group, who collectively care for the OpenGitOps project
+- Ensure the promotion of continued open involvement for the working group
+- Ensure that meetings and other activities are conducted and progress continues to be made against the project agenda, while also engaging other group members in leadership roles
+- Ensure discussion is extended asynchronously to be inclusive of members who cannot attend a specific meeting time
+- Schedule discussion of proposals that have been submitted
+- Partner with Maintainers and other WG members to establish a roadmap and manage ongoing projects
+- Ask for new proposals to be made to address an identified needs
+- Have GitOps domain knowledge
+- Engage deeply in the work being done by the working group - that is, the role is not purely administrative.
+  Chairs are, however not responsible for performing all of the work - this is shared across the working group members and expected from those who have volunteered onto various workstreams.
+- When necessary, handle human and technical coordination across different working group workstreams
+- Handle Code of Conduct violations
+- Resolve escalated decisions when Maintainers can not reach consensus
 
-The OpenGitOps oversight committee is responsible for the overall project, and anything not easily managed by the Maintainers of each git repository. Including:
+Ultimately the chairs - after consulting with the collective of Maintainers and their community - drive the direction, values and governance of the overall project.
 
-- Overseeing the project health and growth
-- Maintaining the brand, mission, vision, values, and scope of the overall project
-- Changes to licensing and intellectual property
-- Administering access to all project assets
-- Administering git repositories as needed
-- Handling Code of Conduct violations
-- Managing financial decisions
-- Defining the scope of each git repository
-- Resolving escalated decisions when Maintainers responsible are blocked
+The GitHub `@gitops-working-group/chairs` team will be kept up to date with current chairs.
 
-Ultimately the committee - after consulting with the collective of Maintainers and their community - drive the direction, values and governance of the overall project.
+## Elections
 
-This committee will initially be comprised of GitOps Working Group Maintainers who have steered the project prior to this initial Governance document.
-The aspiration is no one company or organization should have oversight of the overall project, however that is not yet realistic at this stage. The goal is to broaden maintainership to include a wider range of organizations during CNCF incubation.
+[Chairs](#chairs) will be elected for one year terms and may run for reelection.
```

Comment:
Any thoughts about max # of consecutive terms for an individual?

Reply:
@csand-msft LIFETIME!!! 👑 lol j/k. No strong opinion on this one 🙂

My main concern is to keep this as simple as possible, while learning what we can from other CNCF project governance docs. Ultimately whatever's best for the stability, fairness, and momentum of the group and project.

Personally my gut feeling is setting max terms isn't necessary, since requiring reelection annually should self-govern and weed out anything weird. WDYT?

Reply:
We can also continue to iterate on governance after merging this PR.